lib/src/main/java/com/yalantis/contextmenu/lib/ContextMenuDialogFragment.java

```diff
@@ -49,25 +49,27 @@ public static ContextMenuDialogFragment newInstance(int actionBarSize, List<Menu
     }
 
     @Deprecated
-    public static ContextMenuDialogFragment newInstance(int actionBarSize, List<MenuObject> menuObjects, int animationDelay, int animationDuration) {
+    public static ContextMenuDialogFragment newInstance(int actionBarSize, List<MenuObject> menuObjects, int animationDelay, int animationDuration, int hideAnimationDuration) {
         MenuParams params = new MenuParams();
         params.setActionBarSize(actionBarSize);
         params.setMenuObjects(menuObjects);
         params.setAnimationDelay(animationDelay);
-        params.setAnimationDuration(animationDuration);
+        params.setShowAnimationDuration(animationDuration);
+        params.setHideAnimationDuration(hideAnimationDuration);
         return newInstance(params);
     }
 
     @Deprecated
     @TargetApi(Build.VERSION_CODES.ICE_CREAM_SANDWICH)
     public static ContextMenuDialogFragment newInstance(int actionBarSize, List<MenuObject> menuObjects,
-                                                        int animationDelay, int animationDuration,
+                                                        int animationDelay, int animationDuration, int hideAnimationDuration,
                                                         boolean fitsSystemWindow, boolean clipToPadding) {
         MenuParams params = new MenuParams();
         params.setActionBarSize(actionBarSize);
         params.setMenuObjects(menuObjects);
         params.setAnimationDelay(animationDelay);
-        params.setAnimationDuration(animationDuration);
+        params.setShowAnimationDuration(animationDuration);
+        params.setHideAnimationDuration(hideAnimationDuration);
```

Comment:
> Please also make a constructor without hideAnimationDuration, so it will set by default. Backward compatibility.
My bad -_-"

- fixed mistake with changed deprecated constructor's params. We don't need 2 new deprecated constructors, we just need Backward compatibility.
- corrected params declarations in MenuObject.